File: overflow-wrap.java
public void convertHTMLtoPDF() throws IOException {
    ConverterProperties properties = new ConverterProperties();
    FontProvider fontProvider = new DefaultFontProvider(true, true, true);
    properties.setFontProvider(fontProvider);
    HtmlConverter.convertToPdf(new FileInputStream("HTML/input_html.html"), new FileOutputStream("output_file.pdf"), properties);
}

File: overflow-wrap.cs
public virtual void ConvertHTMLtoPDF() {
    ConverterProperties properties = new ConverterProperties();
    FontProvider fontProvider = new DefaultFontProvider(true, true, true);
    properties.SetFontProvider(fontProvider);
    HtmlConverter.ConvertToPdf(new FileStream("HTML/input_html.html", FileMode.Open, FileAccess.Read), new FileStream("HTML/input_html.html", FileMode.Create), properties);
}
